src/puppetlabs/puppetdb/config.clj

```diff
@@ -358,6 +360,14 @@
             (when dashboard-redirect?
               {:puppetlabs.puppetdb.dashboard/dashboard-redirect-service "/"}))))
 
+(defn- add-mq-defaults
+  [config-data]
+  (-> config-data
+      (update-in [:command-processing :mq :address]
+                 #(or % "vm://localhost?jms.prefetchPolicy.all=1&create=false"))
```

Comment:
Because we needed the bare endpoint value elsewhere, but I can make both vars if we prefer.